api/clients/codecs/eval_poly.go

```diff
@@ -0,0 +1,46 @@
+package codecs
+
+import (
+	"fmt"
+
+	"github.com/Layr-Labs/eigenda/encoding/fft"
+	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
+)
+
+// evalPoly is a polynomial in evaluation form.
+//
+// The underlying bytes represent 32 byte field elements, and the field elements represent the polynomial evaluation
+// at roots of unity.
+type evalPoly struct {
+	fieldElements []fr.Element
+}
```

Comment:
Particular reason why you use a struct instead of `type fieldElements []fr.Element`?

Reply:
Type removed entirely